src/Sylius/Bundle/AdminBundle/templates/product/show/content/page_body/attributes.html.twig

```diff
@@ -0,0 +1,54 @@
+{% set product = hookable_metadata.context.resource %}
+
+<div class="card mb-3">
+    <div class="card-header">
+        <div class="card-title">
+            {{ 'sylius.ui.attributes'|trans }}
+        </div>
+    </div>
+    <div class="card-body" {{ sylius_test_html_attribute('attributes') }}>
+        <div class="bg-white mb-3">
+            <div class="accordion" id="attribute-translations">
+                {% set setLocales = [] %}
+                {% for attribute in product.attributes %}
+                    {% if attribute.localeCode not in setLocales %}
+                        {% set locale = attribute.localeCode %}
+                            <div class="accordion-item">
+                                <h2 class="accordion-header">
+                                    <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#translation-{% if attribute.localeCode %}{{ locale }}{% else %}non-translatable{% endif%}" aria-expanded="{% if loop.first %}true{% else %}false{% endif %}" aria-controls="translation-{% if attribute.localeCode %}{{ locale }}{% else %}non-translatable{% endif %}">
```

Comment:
As there are a lot of code checks, maybe divide this code into the one for the translatable attribute and the non-translatable one